r/extra-tests/test-read-parquet.R

```diff
@@ -0,0 +1,50 @@
+library(arrow)
+library(testthat)
+
+skip_if_version <- function(version, msg, op = `<=`) {
```

Comment:
This `op` is a little non-obvious. I'd rather have explicit skip function names (skip_if_version_equal, skip_if_version_less_than, etc.)